addons/mail/security/ir.model.access.csv

```diff
@@ -31,7 +31,7 @@ access_mail_tracking_value_portal,mail.tracking.value.portal,model_mail_tracking
 access_mail_tracking_value_user,mail.tracking.value.user,model_mail_tracking_value,base.group_user,0,0,0,0
 access_mail_tracking_value_system,mail.tracking.value.system,model_mail_tracking_value,base.group_system,1,1,1,1
 access_publisher_warranty_contract_all,publisher.warranty.contract.all,model_publisher_warranty_contract,,1,1,1,1
-access_mail_template,mail.template,model_mail_template,base.group_user,1,0,0,0
+access_mail_template,mail.template,model_mail_template,base.group_user,1,1,1,0

```

Comment:
hum, I really really don't think it's a good idea

With this anyone can change the template "reset my password" and add a phishing URL in it e.g.

Reply:
We could create a record rule so they user not part of the mail editor group can only create/edit their own template, based on the create_uid